includes/class-wp-document-revisions.php

```diff
@@ -0,0 +1,2118 @@
+<?php
+/**
+ * Main class for WP Document Revisions
+ *
+ * @since 3.0.0
+ * @package WP_Document_Revisions
+ */
+
+/**
+ * Main WP_Document_Revisions class
+ */
+class WP_Document_Revisions extends HTTP_WebDAV_Server {
```

Comment:
Renaming this class is a breaking change.  If intentional, it should be documented as such.